homeassistant/components/sensor/nederlandse_spoorwegen.py

```diff
@@ -135,13 +135,21 @@ def device_state_attributes(self):
             'departure_delay':
                 self._trips[0].departure_time_planned !=
                 self._trips[0].departure_time_actual,
+            'departure_platform':
+                self._trips[0].trip_parts[0].stops[0].platform,
+            'departure_platform_changed':
+                self._trips[0].trip_parts[0].stops[0].platform_changed,
```

Comment:
What does changed mean here? Is it if the platform has changed from their original platform or is it a timestamp when it was last updated? (because the latter should never be stored in state attributes)

Reply:
It is the first: it will be either true or false based upon the api answer. It is not a time stamp.